test/utils/mem-store.js

```diff
@@ -17,44 +19,56 @@ const createMultihash = (data, hashAlg) => {
   })
 }
 
-// const LRU = require('lru')
-// const ImmutableDB = require('./immutabledb-interface')
-// const createMultihash = require('./create-multihash')
+const transformLinksIntoCids = (data) => {
```

Comment:
We probably want to move this to utils.js

Reply:
Hmm, this is not needed anywhere else for now. Perhaps we shall move it if it’s necessary in more than one place?

Reply:
Works for me.